requirements.txt

```diff
@@ -0,0 +1,6 @@
+jupyter_core>=4.6.0
```

Comment:
Let's move all the dependencies to `requirements.txt` file - it's easy to update and use instead of running `setup.py` each time